k8s/migratev2/templates/staging.yaml

```diff
@@ -6,7 +6,6 @@ metadata:
   type: .Values.storageType
 spec:
   storageClassName: {{ .Values.storageClassName }}
-  persistentVolumeReclaimPolicy: Delete
```

Comment:
this should be the default, that's right. I wanted to make it explicit that we relinquish the saving area back to the cluster after we're done.
the actual behavior is pretty  much provider-dependent, hence the explicit parameter here.
See also https://kubernetes.io/docs/concepts/storage/persistent-volumes/#delete

Reply:
reverted